contracts/AlephZero_SpokePool.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+// Arbitrum only supports v0.8.19
+// See https://docs.arbitrum.io/for-devs/concepts/differences-between-arbitrum-ethereum/solidity-support#differences-from-solidity-on-ethereum
+pragma solidity ^0.8.19;
+
+import "./SpokePool.sol";
+import "./libraries/CircleCCTPAdapter.sol";
+import { CrossDomainAddressUtils } from "./libraries/CrossDomainAddressUtils.sol";
+import { ArbitrumL2ERC20GatewayLike } from "./interfaces/ArbitrumBridge.sol";
+
+/**
+ * @notice AVM specific SpokePool. Uses AVM cross-domain-enabled logic to implement admin only access to functions.
+ * @custom:security-contact bugs@across.to
+ */
+contract AlephZero_SpokePool is SpokePool, CircleCCTPAdapter {
```

Comment:
I would rather this be `AlephZero_SpokePool is Arbitrum_SpokePool` this diff should be minimal unless there are required changes from the core Arbitrum functionality, see https://github.com/across-protocol/contracts/blob/93b66791a808ff1634e6ae396f8e362e3216ab86/contracts/WorldChain_SpokePool.sol#L12